metabase/metabase
# Include sandboxing data in the permissions graph change request

## Changes

This PR includes **Group Table Access Policies or GTAP** in the permissions graph request to avoid having orphaned policies. Previously, users could change a table permission to "Sandboxed", create a GTAP which did not wait the permissions graph saving and then just cancel the permissions changes. 

## How to verify

- Run Metabase EE
- Go to the [data permissions](http://localhost:3000/admin/permissions/data/) -> All Users -> Sample Database
- Change the Accounts table permission to Sandboxed, add any mapping, and click Save
- Ensure header with Save and Cancel buttons has appeared, click Save
- Refresh the page and ensure the permission is sandboxed with the correct mapping
- Open the sandboxing settings for the Accounts table and ensure they are exactly as you set them
